### DISCUSSION TITLE
Docker image 👍

### DISCUSSION THREAD

**Discussion by henkiewie** [philippe44/AirConnect#455] [General]
Just a shoutout.
The docker image worked out of the box on mi PI4.
All my Chromecast devices show up and play through the music app.
Except for grouped devices.
Thanks.

**Comment by rajdev19** [comment]
Group devices work as well using the docker image.

**Reply by henkiewie** [reply]
(six months after)... Yes they do now. I had problems changing the volume. There is a delay and they seem to be overruled. It works better from within Android. Have to dive deeper into the settings for groups and syncing is not the best, but I think its all described in the manual.
